### PR TITLE
Entity mappings

### PR DESCRIPTION
**Pull request by Runemoro**

### CODE REVIEW

File: mappings/net/minecraft/entity/ai/brain/task/CompositeTask.mapping [hunk 1]
```diff
@@ -3,6 +3,7 @@ CLASS ajr net/minecraft/entity/ai/brain/task/CompositeTask
 		FIELD c consumer Ljava/util/function/Consumer;
 		METHOD a apply (Lalj;)V
 			ARG 1 list
+	CLASS net/minecraft/entity/ai/brain/task/CompositeTask$b RunMode
```
Comment (Gegy) [inline mappings/net/minecraft/entity/ai/brain/task/CompositeTask.mapping]: Knit bug. Please fix this 😅

Comment (Runemoro) [reply mappings/net/minecraft/entity/ai/brain/task/CompositeTask.mapping]: Fixed

**Review by modmuss50 — APPROVED** [review]

**Comment by modmuss50** [comment]
Looks good to me, ill fix merge conflicts before the update if required.

**Comment by Runemoro** [comment]
Fixed them

**Comment by modmuss50** [comment]
Great, thank you. I will leave this open for either someone else to approve, if not ill merge when we update.

**Pull request merged**